package.json

```diff
@@ -46,8 +50,7 @@
     ]
   },
   "devDependencies": {
-    "@craco/craco": "^7.1.0",
-    "eslint-config-prettier": "^8.9.0",
+    "@craco/craco": "^5.9.0",
```

Comment:
Is there a reason to use this older version of the library?

Reply:
Most of these changes were removed long before I pushed. My current code looks like the red that it stated I removed. Expect for the "react-redux" which I was thinking about using but never actually used it in the code. I will try pushing again.